components/suggestion/suggestion_box.jsx

```diff
@@ -194,6 +195,14 @@ export default class SuggestionBox extends React.Component {
         EventEmitter.removeListener('mention_key_click', this.handleMentionKeyClick);
     }
 
+    calculateInputRect = () => {
+        if (this.inputRef.current) {
+            const rect = ReactDOM.findDOMNode(this.inputRef.current).getBoundingClientRect();
```

Comment:
Instead of using `ReactDOM` here, you might be able to use `this.inputRef.current.getInput()`